examples/zkapps/10-account-updates/src/main.ts

```diff
@@ -67,7 +67,7 @@ import { showTxn, saveTxn, printTxn } from 'mina-transaction-visualizer';
   await showTxn(txn1, 'txn1', legend);
```

Comment:
Hi @barriebyron ,

While testing this code I found, this line is taking alot of time and for me it didn't even completed after 30 mins.
This line is basically being used to generate the images of flow of transactions. The package was created by Evan. 
So, can you please help to check with the MF or O1labs team if we can comment this line ?

Reply:
@LuffySama-Dev  which line do we want to comment out?

Reply:
Hi @barriebyron ,

The lines which I am mentioning will help to generate the images that we have already added in the tutorials.
Images with name: `deploy_txn.png` ,  `txn1-copy.png`, `txn1.png`, `txn2.png`

Commenting these lines will allow the code to run completely, but as Greg said in discord chat this is supposed to be a explainer tutorial, so I am confused like is user also suppose to run this lines to generate images or we can comment and keep them so if users wants to try to generate image they can uncomment and try it (For me it didn't worked).

The line numbers are 
```
67:   await showTxn(txn1, 'txn1', legend);
68:   await saveTxn(txn1, 'txn1', legend, './txn1.png');

80:   await showTxn(txn2, 'txn2', legend);
81:    await saveTxn(txn2, 'txn2', legend, './txn2.png');
```

Thank You 😃

Reply:
If the visualizer doesn't work, let's move forward without it. good plan, thank you

Reply:
> @LuffySama-Dev which line do we want to comment out? if you comment out and run, are the results good? Thanks for asking in Discord https://discord.com/channels/484437221055922177/1209120015530463304/1209120015530463304
> 
> Update: We do not have access to the code author... can you update the tutorial to omit the visualizer and the images?

We will still need Images just in case someone wants to see in example and I can comment out the code lines. Can I proceed and push the code ?
cc: @barriebyron